src/saving/SaveManagerGUI.tscn

```diff
@@ -223,6 +224,24 @@ __meta__ = {
 "_edit_use_anchors_": false
 }
 
+[node name="SaveDirectoryWarningDialog" type="AcceptDialog" parent="CenterContainer/VBoxContainer"]
+visible = true
+margin_top = 516.0
+margin_right = 1000.0
+margin_bottom = 600.0
+window_title = "NO_SAVE_DIRECTORY"
+
+[node name="DialogText" type="Label" parent="CenterContainer/VBoxContainer/SaveDirectoryWarningDialog"]
+margin_left = 10.0
+margin_top = 10.0
+margin_right = 990.0
+margin_bottom = 32.0
+text = "TRY_NEW_GAME"
```

Comment:
I think this should be named more accurately, something like `TRY_MAKING_A_SAVE`
as that's what creates the folder and not starting a new game...

Reply:
No worries. Fixed.